Resources/Prototypes/Entities/Structures/Storage/Closets/Lockers/lockers.yml

```diff
@@ -359,6 +359,19 @@
     stateBaseClosed: shotguncase
     stateDoorOpen: shotguncase_open
     stateDoorClosed: shotguncase_door
+  - type: AccessReader
+  - type: Construction
+    graph: GunSafeBase
+    node: done
+    containers:
+    - entity_storage
+  - type: StaticPrice
+    price: 335
+
+- type: entity
+  id: GunSafeBaseSecure
+  parent: GunSafeBase
+  components:
```

Comment:
add a suffix `Armory` or something, so in spawn menu you could see the difference